<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMapDialog</class>
 <widget class="QDialog" name="AddMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Map</string>
  </property>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Map</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHeight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtWidth">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Width (in tiles):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (in tiles):</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstTilesets">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tileset:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
